action.yml

```diff
@@ -40,9 +40,17 @@ inputs:
     description: 'create or destroy'
     required: false
     default: 'create'
+  USE_PULUMI:
+    description: 'use pulumi instead of terraform'
+    required: false
+    default: 'false'
   CREATE_ACR:
     description: 'create ACR with cluster name'
     default: 'false'
+  PULUMI_ACCESS_TOKEN:
+    description: 'create ACR with cluster name'
```

Comment:
is the description and default correct for this?

Reply:
no it's wrong. I'll update